omero-installation/omero_installation_debian8_ice35.md

```diff
@@ -0,0 +1,162 @@
+### OMERO.server installation on Debian 8 and Ice 3.5
+
+This is an example walkthrough for installing OMERO 5.2.x on Debian 8, using a dedicated system use.
+These instructions assume your Linux distribution is configured with a UTF-8 locale.
+
+#### Name: OMERO team
+
+#### Institute: University of Dundee
+
+#### Contact: OME External Developer List <ome-devel@lists.openmicroscopy.org.uk>
+
+#### Context:
+
+For convenience in this walkthrough the main OMERO configuration options have been defined as environment variables. When following this walkthrough you can either use your own values:
+ 
+  	OMERO_DB_USER=db_user
+	OMERO_DB_PASS=db_password
+	OMERO_DB_NAME=omero_database
+	OMERO_ROOT_PASS=omero_root_password
+	OMERO_DATA_DIR=/OMERO
+
+	OMERO_WEB_PORT=80
+
+	export OMERO_DB_USER OMERO_DB_PASS OMERO_DB_NAME OMERO_ROOT_PASS OMERO_DATA_DIR OMERO_WEB_PORT
+
+	export PGPASSWORD="$OMERO_DB_PASS"
+
+#### System Specifications:
+
+Debian 8 with Java 1.8, Ice 3.5 and PostgreSQL 9.4. Nginx 1.8 for OMERO.web
+
+#### Installation:
+
+The following steps describe how to install an OMERO.server and to deploy OMERO.web
+using Nginx.
+
+##### Step 1:
+
+As **root**, Install:
+
+Java 1.8:
+
+	echo 'deb http://httpredir.debian.org/debian jessie-backports main' > /etc/apt/sources.list.d/jessie-backports.list
+	apt-get update
+	apt-get -y install openjdk-8-jre-headless
+
+Various dependencies:
+	
+	apt-get -y install \
+	python-{matplotlib,numpy,pip,scipy,tables,virtualenv}
+
+	apt-get -y install \
+	libtiff5-dev \
+	libjpeg62-turbo-dev \
+	zlib1g-dev \
+	libfreetype6-dev \
+	liblcms2-dev \
+	libwebp-dev \
+	tcl8.6-dev \
+	tk8.6-dev
+
+	pip install --upgrade pip
+
+Pillow: 
+	
+	pip install Pillow<3.0
```

Comment:
FWIW rather than use pip directly I seem to have `python-pil` installed (it's 2.6.1). Might simplify the walkthrough.